tests/python/train/test_resnet_aug.py

```diff
@@ -39,14 +40,15 @@ def get_net():
     return softmax
 
 # check data
-get_cifar10()
+path = tempfile.mkdtemp()
```

Comment:
You're not cleaning up these temporary directories. You could use pytests functionality which will handle cleanup https://docs.pytest.org/en/latest/tmpdir.html

Reply:
thanks. this is useful. I will replace other occurrences.